[bug] Releases 0.0.8 will throw an error when starting server

npm create vite@latest

use the plugin 

npm run dev

`Cannot find module '/***/node_modules/rollup-plugin-react-scoped-css/dist/ast-program' imported from /***/node_modules/rollup-plugin-react-scoped-css/dist/index.js`


and it's correct in Releases 0.0.7

--- a/package.json
+++ b/package.json
@@ -4,7 +4,6 @@
   "description": "A rollup plugin designed to allow scoped css to be run in react (Compatible with vite and rollup)",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
-  "type": "module",
   "scripts": {
     "build": "tsc",
     "build:watch": "tsc --watch",

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "target": "ES2021",
-    "module": "ESNext",
+    "module": "CommonJS",
     "moduleResolution": "node",
     "declaration": true,
     "outDir": "dist",

